src/parser/jobs/gnb/modules/Continuation.tsx

```diff
@@ -0,0 +1,89 @@
+import {Trans} from '@lingui/react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {AbilityEvent} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import Checklist, {Requirement, TARGET, TieredRule} from 'parser/core/modules/Checklist'
+
+const SKILLS_TO_WATCH = [
+	ACTIONS.GNASHING_FANG.id,
+	ACTIONS.SAVAGE_CLAW.id,
+	ACTIONS.WICKED_TALON.id,
+	ACTIONS.JUGULAR_RIP.id,
+	ACTIONS.ABDOMEN_TEAR.id,
+	ACTIONS.EYE_GOUGE.id,
+]
+
+const WARN_TARGET_PERCENT = 99 // Missing even 1 should qualify as a warning.
+
+// TODO: Figure out how to check if the next Continuation was even usable before the fight ended.
+export default class Continuation extends Module {
+
+	@dependency private checklist!: Checklist
+
+	private ripBuffCounter = 0
+	private tearBuffCounter = 0
+	private gougeBuffCounter = 0
+	private ripCounter = 0
+	private tearCounter = 0
+	private gougeCounter = 0
+
+	protected init() {
+		this.addHook('cast', {by: 'player'}, this.onTrigger)
```

Comment:
Given you're summing the values at the end, you can probably simplify this handling such that it only hooks casts it needs to know about. Something _vaguely_ along the lines of...

```js
ACTIONS = [RIP, TEAR, GOUGE]
STATUSES = [READY_TO_RIP, READY_TO_TEAR, READY_TO_GOUGE]

this.addHook('cast', {by: 'player', abilityId: ACTIONS}, () => this.actions++)
this.addHook('applybuff', {by: 'player', abilityId: STATUSES}, () => this.buffs++)
```

(also saves the switch)